catkin_ws/src/image_rotation/src/image_algorithm.cpp

```diff
@@ -0,0 +1,23 @@
+///
+/// @file
+/// @brief Implementation of image processing class
+///
+
+#include "ImageAlgorithm.h"
+
+using namespace cv;
+
+void ImageAlgorithm::RotateImage(Mat &image, double angle)
+{
+    // get rotation matrix for rotating the image around its center in pixel coordinates
+    Point2f center((image.cols - 1) / 2.0, (image.rows - 1) / 2.0);
+    Mat rotation_matrix = getRotationMatrix2D(center, angle, 1.0);
+    // determine bounding rectangle, center not relevant
+    Rect2f bbox = RotatedRect(Point2f(), image.size(), angle).boundingRect2f();
```

Comment:
change variable name from bbox to bounding_box

Reply:
changed